helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -102,8 +102,7 @@
 import static org.apache.helix.HelixConstants.ChangeType.RESOURCE_CONFIG;
 import static org.apache.helix.HelixConstants.ChangeType.TARGET_EXTERNAL_VIEW;
 
-
-@PreFetch(enabled = false)
+@PreFetchChangedData(enabled = false)
```

Comment:
Previously (in production) callbackhandler not using prefetch, do we verify?

Reply:
@kaisun2000 Double verify here: https://github.com/apache/helix/issues/1331#issuecomment-683212819